Sources/HTTP/HTTPServer.swift

```diff
@@ -28,21 +28,26 @@ public protocol HTTPServing : class {
 public class HTTPServer: HTTPServing {
     private let server = PoCSocketSimpleServer()
 
+    /// Create an instance of the server. This needs to be followed with a call to `start()`
     public init() {
     }
 
+    /// Start the HTTP server on the given `port`, using `handler` to process incoming requests
```

Comment:
I would add `- Parameter` too for port and handler, what do you think?

Reply:
I saw that in some other place of the code it's missing, if it's okay I would raise a PR fixing the missing docs for parameters.